docs/pages/management/export-audit-events/elastic-stack.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Monitor Teleport Audit Events with the Elastic Stack"
+title: "Export Teleport Audit Events with the Elastic Stack"
```

Comment:
Logstash is exporting audit events while Elasticsearch is ingesting/storing/indexing them. Is there another way to describe this?

Reply:
I will again refer to @zmb3 who suggested this term for all these docs.

Reply:
Maybe "Export ... to the Elastic Stack"?